Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -69,11 +69,17 @@ public class RocksDBStore : BaseBlockStatesStore
         /// <param name="blockCacheSize">The capacity of the block cache.</param>
         /// <param name="txCacheSize">The capacity of the transaction cache.</param>
         /// <param name="statesCacheSize">The capacity of the states cache.</param>
+        /// <param name="maxTotalWalSize">The number to configure max_total_wal_size RocksDB option.
+        /// </param>
+        /// <param name="keepLogFileNum">The number to configure keep_log_file_num RocksDB option.
+        /// </param>
```

Comment:
Added parameters should be recorded in the changelog (with the prefix <q>(Libplanet.RocksDBStore)</q>).

```suggestion
        /// <param name="maxTotalWalSize">The number to <c>configure max_total_wal_size</c> RocksDB
        /// option.</param>
        /// <param name="keepLogFileNum">The number to configure <c>keep_log_file_num</c> RocksDB
        /// option.</param>
```

Reply:
@moreal Please check this too.

Reply:
I applied it in 6f502660ee2b9a893176e53fa5acf128c4bb88b5